.github/workflows/testing.yml

```
@@ -33,12 +33,18 @@ jobs:
           VALIDATE_NATURAL_LANGUAGE: false
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Get tag
+        id: get_tag
+        run: |
+          echo "::set-output name=tag::$(./ci/get-latest-tag.sh)"
```

I think this could be shorter
```suggestion
          VERSION="$(git tag | grep -v kangal | sort -rV | head -n 1)"
          echo "::set-output name=tag::$VERSION"
```